seisflows/preprocess/default.py

```diff
@@ -28,9 +28,16 @@ class Default:
 
     Parameters
     ----------
-    :type data_format: str
-    :param data_format: data format for reading traces into memory. For
-        available see: seisflows.plugins.preprocess.readers
+    :type syn_data_format: str
```

Comment:
`syn_data_format` is shared by preprocess.default and solver.specfem and both classes list `syn_data_format` in their class docstring, which causes it to show up twice in the dynamically generated parameter file created during `seisflows configure`.

```yaml
#
#    Solver SPECFEM
#    --------------
#    Defines foundational structure for Specfem-based solver module.
#    Generalized SPECFEM interface to manipulate SPECFEM2D/3D/3D_GLOBE w/ Python
#
#    Parameters
#    ----------
#    :type syn_data_format: str
#    :param syn_data_format: data format for reading synthetic traces into memory.
#        Available: ['SU': seismic unix format, 'ASCII': human-readable ascii]
...
#    Default Preprocess
#    ------------------
#    Data processing for seismic traces, with options for data misfit,
#    filtering, normalization and muting.
#
#    Parameters
#    ----------
#    :type syn_data_format: str
#    :param syn_data_format: data format for reading synthetic traces into
#        memory. Shared with solver module. Available formats: 'su', 'ascii'
```


Could you move one of the docstrings for `syn_data_format` into the __init__() function of its respective class. This allows it to still be available in the source code, but to not show up twice in the parameter file.

I would suggest moving this one, since `syn_data_format` is more relevant to the solver, but I leave it up to you!